bundles/python/python_lexer.moon

```diff
@@ -46,12 +46,16 @@ howl.aux.lpeg_lexer ->
   operator = c 'operator', S'+-*/%~&^=!<>;:,.(){}[]|`'
 
   name = (alpha + '_')^1 * (alpha + digit + P'_')^0
+  type_name = c 'type', (upper^1 * (alpha + digit + '_')^0)
```

Comment:
I don't know about this...a lot of Python scripts will begin functions with uppercase letters. In particular, I know a lot of Python scripts, including [several within Chromium](https://chromium.googlesource.com/chromium/src/build/+/master/gn_helpers.py) and [SCons](https://bitbucket.org/scons/scons/src/1e0827962a2bb114041df47d44be2384c86f3918/examples/src_packaging/SConstruct), use PascalCase for functions.